samples/DotNet/Microsoft.Azure.EventHubs/SampleSender/readme.md

```diff
@@ -1,16 +1,15 @@
 # Send events to Azure Event Hubs in .NET Standard
 
-This sample shows how to write a .NET Core console application that sends a set of events to an Event Hub. You can run the solution as-is, replacing the `EhConnectionString` and `EhEntityPath` strings with your Event Hub values. The sample is also [available as a tutorial](https://docs.microsoft.com/azure/event-hubs/event-hubs-dotnet-standard-getstarted-send).
```

Comment:
This link gets redirected to Track 2 quick start, so re-worded the entire intro to refer to the link as T2 and not T1